test/Api/CatalogFixtures.php

```diff
@@ -0,0 +1,245 @@
+<?php
+namespace SquareConnect\Api;
+
+class CatalogFixtures extends \PHPUnit_Framework_TestCase
```

Comment:
All the other tests have it. Any particular reason to keep here or remove everywhere else?